checking_class.rb

```diff
@@ -0,0 +1,55 @@
+module Bank
+  class Checking < Account
+
+    def initialize(balance)
+      super(balance)
+      @check_count = 0
+    end
+
+    #method for withdrawal
+    def withdrawal(subtract_money)
+      if (@balance - subtract_money - 1) >= 0
+        @balance = (@balance - subtract_money - 1)
+        #returns updated account balance
+        return @balance
+      else
+      #rejects negative balance, and returns current balance
+        puts "Your withdrawal was rejected. You only have #{@balance} dollars."
+        puts "You are not allowed to overdraft. Try a smaller withdrawal."
+      end
+    end
+
+
+    def withdraw_using_check(subtract_money)
+      if (@balance - subtract_money.abs) >= -10
+        @balance = (@balance - subtract_money.abs)
+        #returns updated account balance with a transaction fee of $1 removed from balance
+        #check_count = 0
+        @check_count += 1
+          if @check_count > 3
```

Comment:
Watch your indentation here, you are indented one extra level that you don't need